gen-val/src/crypto/src/NIST.CVP.ACVTS.Libraries.Crypto.Common/Asymmetric/XECDH/Enums/XecdhEnums.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Runtime.Serialization;
+
+namespace NIST.CVP.ACVTS.Libraries.Crypto.Common.Asymmetric.XECDH.Enums
+{
+    public enum Curve
```

Comment:
Might need a different name to not be confused with ECDSA curves. Should probably rename both the more I think about it.

Reply:
Currently there's the following files:
- gen-val/src/crypto/src/NIST.CVP.ACVTS.Libraries.Crypto.Common/Asymmetric/DSA/ECC/Enums/EccEnums.cs
- gen-val/src/crypto/src/NIST.CVP.ACVTS.Libraries.Crypto.Common/Asymmetric/DSA/Ed/Enums/EdEnums.cs
- gen-val/src/crypto/src/NIST.CVP.ACVTS.Libraries.Crypto.Common/Asymmetric/XECDH/Enums/XecdhEnums.cs

All of them define a Curve enum so they are disambiguated by importing the correct file.